scripts/rpc-maas-tool.py

```diff
@@ -671,8 +671,9 @@ def _get_failed_checks(self, checks=None):
                 try:
                     result = self.rpcm.conn.test_existing_check(check)
                 except rackspace.RackspaceMonitoringValidationError as e:
-                    validation_error = (" Validation Error: %(s):"
-                                        % {'e': e.message})
+                    validation_error = " Validation Error: {s}:".format(
+                        s=e.message
+                    )
                     break
```

Comment:
Added rcbops/u-suk-dev#1532 to address the wider logic issues. This can still be merged to fix the key error.